### DISCUSSION TITLE
Add `Export as SVG` option in db.build

### DISCUSSION THREAD

**Discussion by Ebrahim-Ramadan** [supabase/supabase#32581] [Feature Requests]
the [db.build](https://db.build) service is great, but devs may get to craft a schema for preview or quick docs, so a `Export as SVG` button, that creates pdf or svg page of the tables and their relations view, might just be a great feature

**Comment by Hallidayo** [comment]
Hi @Ebrahim-Ramadan, thank you for opening this issue. As this is a feature request I'm going to move the issue over to discussions/enhancements but again thank you for opening.
